graphql_compiler/tests/integration_tests/test_backends_integration.py

```diff
@@ -124,6 +124,23 @@ def test_simple_filter(self, backend_name):
 
         self.assertResultsEqual(graphql_query, parameters, backend_name, expected_results)
 
+    @integration_fixtures
+    def test_colocated_filter_and_tag(self):
+        graphql_query = '''{
+            Animal {
+                name @output(out_name: "animal_name")
```

Comment:
There's already a `colocated_filter_and_tag` test input in the test inputs file. Mind syncing up this query and that one so they are exactly identical, and updating any tests that are affected?

Reply:
Also, the generated MATCH for `test_colocated_filter_and_tag()` is incorrect -- it is missing the `Animal` type bound. I'm guessing that the author of the test didn't write it by hand, but just ran the code and copy-pasted the output into the "expected output" field. Let's try to not do that, because this is how we get bitten when we do.